docs/mast/mast.rst

```diff
@@ -181,6 +180,111 @@ To get a table of metadata associated with observation or product lists use the
 
 
 
+Mission Searches
+================
+
+Mission-Specific Search Queries
+-------------------------------
+
+These queries allow for searches based on mission-specific metadata for a given
+data collection.  Currently it provides access to a broad set of Hubble Space
+Telescope (HST) metadata, including header keywords, proposal information, and
+observational parameters.  The available metadata includes all information that
+was previously available in the original HST web search form, and are present in
+the current Mission Search interface.
+
+Currenlty, the API only includes the search functionality. The functionality to
+download data products associated with search results is not currently supported. 
+
+An object of MastMissions class is instantiated with a default mission of 'hst' and 
+default service set to 'search'. set_service() and set_mission() can be used to change
+the service and mission to other values but right now, API only supports search service
+and 'hst' mission.
+
+.. code-block:: python
+
+                >>> from astroquery.mast.missions import MastMissions
+                >>> missions = MastMissions()
+                >>> missions.mission
+                'hst'
+                >>> missions.service
+                'search'
+
+The missions object can be used to search meta data using region coordinates. the kwargs can be
+used to specify output characteristics like selec_cols and sort_by and conditions that filter on 
+values like proposal id, pi last name etc. The available column names to filter and
+their descriptions can be found here https://vao.stsci.edu/missionmast/tapservice.aspx/tables#folder38
```

Comment:
meta data -> metadata
the kwards -> The keyword arguments.
Use quotes around actual arguments, like "selec_cols". If "proposal id" and "pi last name" are the actual values, surround by quotes, if not, fix the capitalization/spelling (proposal ID, principle investigator last name).
Turn https://vao.stsci.edu/missionmast/tapservice.aspx/tables#folder38 into an actual link \`here <https://vao.stsci.edu/missionmast/tapservice.aspx/tables#folder38>\`__, is the \#folder38 necessary?